### PR TITLE
test: update documentation in feature_csv_activation.py

### PR DESCRIPTION
**Pull request by PiRK**
These changes in the test documentation reflect the changes introduced in #17921.

### CODE REVIEW

**Review by theStack — APPROVED** [review]
ACK e2a206329fca3158e25e8a7b279e92ec8763a04b
Verified that the numbers match, also the grammar/stylistic changes LGTM.

File: test/functional/feature_csv_activation.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
 BIP 112 - CHECKSEQUENCEVERIFY
 BIP 113 - MedianTimePast semantics for nLockTime
 
-mine 82 blocks whose coinbases will be used to generate inputs for our tests
-mine 345 blocks and seed block chain with the 82 inputs will use for our tests at height 427
+mine 83 blocks whose coinbases will be used to generate inputs for our tests
+mine 344 blocks and seed blockchain with the 83 inputs used for our tests at height 427
```
Comment (jonatack) [inline test/functional/feature_csv_activation.py:13]: nit, I think there's no need to change "block chain"; see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words

Comment (PiRK) [reply test/functional/feature_csv_activation.py]: I reverted this change.

**Comment by MarcoFalke** [comment]
review ACK  9053b88 

checked that  COINBASE_BLOCK_COUNT is 83. didn't check the other constant

**Pull request merged**